File: api/src/models/HorariosModel.js
/*
    Esquema de para la colección Menus en la base de datos
    En los esquemas deben poner todos los atributos de la base de datos para poder manipularlos.
*/

const mongoose = require('mongoose');
const HorariosSchema = new mongoose.Schema({

    _id: { type: Number},
    id_restaurante: { type: Number, ref: 'Locales' },
    dia_semana: { type: String,
        enum: ['Lunes', 'Martes', 'Miercoles', 'Jueves', 'Viernes', 'Sabado', 'Domingo'],
        required: true
     },
    hora_apertura: { type: String },
    hora_cierre: { type: String },
    cierre_por_ocasion_especial: { type: Boolean}
},{ collection: 'Horarios' });

module.exports = mongoose.model('Horarios', HorariosSchema)

File: web/MongoProyectoWeb/MongoProyectoWeb/Models/HorariosModel.cs
using System.Text.Json.Serialization;

namespace MongoProyectoWeb.Models
{
    public class HorariosModel
    {
        [JsonPropertyName("_id")]
        public int? _id { get; set; }

        [JsonPropertyName("id_restaurante")]
        public int? IdRestaurante { get; set; }

        [JsonPropertyName("dia_semana")]
        public string? Dia_semana { get; set; }

        [JsonPropertyName("hora_apertura")]
        public string? Hora_apertura { get; set; }

        [JsonPropertyName("hora_cierre")]
        public string? Hora_cierre { get; set; }
        [JsonPropertyName("cierre_por_ocasion_especial")]
        public bool Cierre_por_ocasion_especial { get; set; }


    }
}
